### ISSUE TITLE
Navigating to an ohsome dashboard link that has empty tag values (indicating all values) incorrectly gets replaced by "tree" as the value

### ISSUE THREAD

**Issue by watmildon** [GIScience/ohsome-dashboard#3]
I'm tracking work for address additions in [this project](https://tasks.openstreetmap.us/projects/377) by looking at the Count of addr:housenumber tags on all nodes, ways, and relations from the first of 2023 to today (usually by week).

The generated link looks correct:
```
https://dashboard.ohsome.org/#backend=ohsomeApi&groupBy=none&time=2023-01-01T00%3A00%3A00Z%2F2023-07-02T20%3A00%3A00Z%2FP1W&key=addr%3Ahousenumber&value=&types=node%2Cway%2Crelation&measure=count&adminids=-162018
```

but navigating to the page rewrites the link to this incorrect one, with the empty tag value replaced by "tree":
```
https://dashboard.ohsome.org/#backend=ohsomeApi&groupBy=none&time=2023-01-01T00%3A00%3A00Z%2F2023-07-02T20%3A00%3A00Z%2FP1W&key=addr%3Ahousenumber&value=tree&types=node%2Cway%2Crelation&measure=count&adminids=-162018
```

The chart should look something like:
![image](https://github.com/GIScience/ohsome-dashboard/assets/118567155/840bd603-267a-4534-964c-3f87788bd462)

Let me know if there's any other information I can provide.

**Comment by tyrasd** [comment]
Thank you for the detailed report. This is now fixed and already live on the deployed version on https://dashboard.ohsome.org

**Comment by watmildon** [comment]
Happy to help. Thanks for the quick update!

**Comment by mcauer** [comment]
Feedback from Jochen Topf was:
Ich hab grad mal probiert, wenn ich den "value" in der URL bei Euch
weglasse, also so

* https://dashboard.ohsome.org/#backend=ohsomeApi&key=amenity&types=node%2Cway

Dann hat er als value aus irgendeinem Grund "tree" eingesetzt. Das
scheint mir noch ein Fehler zu sein. 


no key/ no value => use defaults
key there / no value => use key set value to empty (meaning *)
key there/ value empty => same as above
key there/ value there => use them

**Comment by tyrasd** [comment]
@mcauer that's a slightly different issue :sweat_smile: (empty, but present `value ` vs. omitted `value`). But I agree that it does not make sense to set the field default values for the `value` form element if a `key` was submitted via the URL. That should be fixed in 7753c98.
